tcg/battle.ts

```
@@ -168,17 +194,46 @@ export class Battle {
 
   /**
    * Append a line to the currently-accumulating action log (and the full turn history).
-   * Call sites: damage resolution, effect application, knockouts, etc.
+   * `text` must be PLAIN (no markdown); `kind` drives per-surface styling.
+   * Call sites: skill/item announcements, damage resolution, effect application, KOs, etc.
    */
-  logEvent(message: string): void {
-    this.currentActionLog.push(message);
-    this.turnHistory.push(message);
-    log(`[tcg] ${message}`);
+  logEvent(text: string, kind: BattleLogKind = 'info', detail?: string): void {
```

ESLint

[error] Default parameters should be last.

(default-param-last)